lib/broadside/configuration.rb

```diff
@@ -57,7 +57,10 @@ def ssh_cmd(ip, options = {})
       cmd << " -i #{@ssh[:keyfile]}" if @ssh[:keyfile]
       if (proxy = @ssh[:proxy])
         raise MissingVariableError, "Bad proxy: #{proxy[:host]}/#{proxy[:port]}" unless proxy[:host] && proxy[:port]
-        cmd << " -o ProxyCommand=\"ssh #{proxy[:host]} nc #{ip} #{proxy[:port]}\""
+        cmd << ' -o ProxyCommand=\"ssh -q '
+        cmd << "-i #{@ssh[:proxy][:keyfile]} " if @ssh[:proxy][:keyfile]
```

Comment:
minor nitpick: i feel like we should stick with the convention of putting the space in the front; otherwise it's easy to muck stuff up later (though the specs will save you)